Docker/README.md

```diff
@@ -77,11 +77,11 @@ Since the [Dockerfile](Dockerfile) is located under a different folder other tha
 it is important to define the context `../` and allow the Dockerfile properly COPY the source
 code to be compiled on later steps.
 
-### Access primary node gRPC endpoint
+### Access primary node public gRPC endpoints
 
 The nodes by default are running with the `Tusk` algorithm disabled, which basically allow
 to user to treat Narwhal as a pure mempool. When that happens, the gRPC server is bootstrapped
-for the primary nodes and that allow someone to interact with the node.
+for the primary nodes and that allow someone to interact with the node (e.x the consensus layer).
```

Comment:
Nit: e.x? Did you mean e.g.?

Reply:
Good point, wanted to go for `ex` , but preserving the dot will push me to use `e.g` :-)